src/main/kotlin/no/nav/meldeplikt/meldekortservice/api/Internal.kt

```diff
@@ -0,0 +1,50 @@
+package no.nav.meldeplikt.meldekortservice.api
+
+import io.ktor.application.call
+import io.ktor.http.ContentType
+import io.ktor.response.respond
+import io.ktor.response.respondRedirect
+import io.ktor.response.respondText
+import io.ktor.routing.Route
+import io.ktor.routing.Routing
+import io.ktor.routing.get
+import io.ktor.routing.route
+import io.ktor.util.KtorExperimentalAPI
+import no.nav.meldeplikt.meldekortservice.config.SWAGGER_URL_V1
+import no.nav.meldeplikt.meldekortservice.config.swagger
+import no.nav.meldeplikt.meldekortservice.utils.API_PATH
+import no.nav.meldeplikt.meldekortservice.utils.BASE_PATH
+import no.nav.meldeplikt.meldekortservice.utils.INTERNAL_PATH
+import no.nav.meldeplikt.meldekortservice.utils.swagger.SwaggerUi
+
+fun Route.healthApi() {
+
+    route(INTERNAL_PATH) {
+
+        get("/isAlive") {
+            call.respondText(text = "Alive", contentType = ContentType.Text.Plain)
+        }
+
+        get("/isReady") {
+            call.respondText(text = "Ready", contentType = ContentType.Text.Plain)
+        }
+
+        get("/ping") {
+            val pingJsonResponse = """{"ping": "pong"}"""
+            call.respondText(pingJsonResponse, ContentType.Application.Json)
```

Comment:
Bør vi ha en standard for om parametre skal navngis eller ikke? (her blir det blanding)

Reply:
Ja, vi burde nok hvertfall ha en liten diskusjon om hvor det er nødvendig å ha det og hvor det ikke trengs. I første omgang legger jeg til `text` og `contentType` her.

Reply:
Enig. Det er ikke sikkert at det alltid er hensiktsmessig, men for like kall som her så bør det være likt (enten med eller uten) i samme fil.